plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/BaseClickHouseConnectorTest.java

```diff
@@ -564,10 +564,16 @@ protected Optional<DataMappingTestSetup> filterDataMappingSmokeTestData(DataMapp
                 // TODO (https://github.com/trinodb/trino/issues/7101) enable the test
                 return Optional.empty();
 
-            case "time":
-            case "time(6)":
             case "timestamp":
             case "timestamp(6)":
+                // TODO: We have mapped Trino `Timestamp(p)` to ClickHouse `Datetime64(p)` and we
```

Comment:
is there some issue tracking this on the JDBC driver side?

Reply:
I have searched before and found nothing, I will spend some time investigating this issue.